src/modules/new-request-form/fields/DropDown.tsx

```diff
@@ -22,12 +22,17 @@ export function DropDown({ field }: DropDownProps): JSX.Element {
         inputProps={{ name, required }}
         isEditable={false}
         validation={error ? "error" : undefined}
+        renderValue={({ selection }) =>
+          selection && "value" in selection
+            ? options.find((option) => option.value === selection.value)?.name
+            : "-"
+        }
       >
         {options.map((option) => (
           <Option
             key={option.value}
             value={option.value}
-            isSelected={option.value === value}
+            isSelected={option.value.toString() === value.toString()}
```

Comment:
> I was testing the CC field, and in that case, we need to pass different values when we submit the form (e.g. request[collaborators][]: test@example.com, request[collaborators][]: test2@example.com) but then we receive back a string value: "test@example.com,  test2@example.com".

I understand what you mean and that's an interesting problem as well!
But is the type still valid? Meaning `Field["value"]` is always a `string`, before and after submission?

Because in the case of a `field` with `options` that is not the case:
- Before submitting the form we have `type Field { value: number }`
- When validation fails, we have `type Field { value: string }`

So if we don't fix this on the API side, we should update the type to:
```ts
type Field { 
  value: number | string
}
```

Reply:
So, now we are getting a `number` at the beginning and then a `string` when validation fails. That's not ideal and as you said it would be better to have the same type.

I was thinking of returning always a string for consistency and simplicity. In the future, we'll need to handle also the number custom field, and if I remember correctly we'll need to use string as well o we will risk losing precision for decimal values